pkgs/tools/misc/mlxbf-bootctl/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mlxbf-bootctl";
+  version = "1.1-6";
+
+  src = fetchFromGitHub {
+    owner = "Mellanox";
+    repo = pname;
+    rev = "${pname}-${version}";
+    hash = "sha256-F49ZZtty+NARXA/doAFLhsQn4XkPW6GWLXGy4waIaM0=";
+  };
+
+  installPhase = ''
+    mkdir -p "$out"/bin
+    cp -a mlxbf-bootctl "$out"/bin/
```

Comment:
should use `install` here, grep the repo for examples

Reply:
Sadly they hardcde $(DESTDIR)/sbin :(
https://github.com/Mellanox/mlxbf-bootctl/blob/master/Makefile#LL2C1-L2C16

Reply:
`install` -D mlxbf-bootctl $out/bin/mlxbf-bootctl` This is what you wanted, right?